multimodal/setup.py

```diff
@@ -33,13 +33,15 @@
     "evaluate>=0.2.2,<0.4.0",
     "accelerate>=0.9,<0.17",
     "timm>=0.6.12,<0.7.0",
-    "torch>=1.9,<1.14",
-    "torchvision<0.15.0",
+    "torch>=1.9,<2.1",
+    "torchvision>=0.10.0,<0.16",
     "fairscale>=0.4.5,<0.4.14",
-    "scikit-image>=0.19.1,<0.20.0",
-    "pytorch-lightning>=1.9.0,<1.10.0",
+    "scikit-image>=0.19.1,<0.21.0",
+    "pytorch-lightning>=1.9.0,<2.1",
     "text-unidecode>=1.3,<1.4",
-    "torchmetrics>=0.11.0,<0.12.0",
+    # temporary for testing ref https://github.com/autogluon/autogluon/issues/2687#issuecomment-1531122312
+    # "torchmetrics>=0.11.0,<0.12.0",
+    "torchmetrics @ https://github.com/Lightning-AI/torchmetrics/archive/95599c9a7bd5e6c8f0a8039562dbbc832b434031.zip",
```

Comment:
@FANGAreNotGnu fyi I've [asked](https://github.com/Lightning-AI/torchmetrics/pull/1625#issuecomment-1533615952) when this commit with pt 2.0 support will be released

Reply:
`1.0.0rc0` is out